modules/UI/UI.js

```diff
@@ -227,8 +231,8 @@ UI.initConference = function () {
 
     APP.store.dispatch(checkAutoEnableDesktopSharing());
 
-    if(!interfaceConfig.filmStripOnly) {
-        Feedback.init(eventEmitter);
+    if (interfaceConfig.filmStripOnly) {
+        APP.store.dispatch(shouldShowPostCallFeedbackDialog(false));
```

Comment:
That makes sense. I've modified the new action to take recorder and filmstrip into account.